onnx/test/shape_inference_test.py

```diff
@@ -8567,7 +8567,7 @@ def test_tree_ensemble_classifier(self) -> None:
                 "TreeEnsembleClassifier",
                 ["x"],
                 ["y", "z"],
-                class_ids=[0, 1, 2, 3, 4],
+                classlabels_int64s=[0, 1, 2, 3, 4],
```

Comment:
Curious what changed here? Is this just an old bug?

Reply:
Oh, I remember ... I guess the shape-inference was added recently to these onnxml operators.

Reply:
class_ids' size is not the number of classes. Instead, either classlabels_strings or classlabels_int64s has a size of the output class numbers.

Reply:
Yes it is an old bug.